misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/JvmStaticMethod.java

```diff
@@ -63,4 +71,40 @@ public Value invoke() throws EvaluationException {
                     .getString(), methodRef.name()));
         }
     }
+
+    @Override
+    protected List<Value> getMethodArgs(JvmMethod method) throws EvaluationException {
+        try {
+            if (argValues == null && argEvaluators == null) {
+                throw new EvaluationException(String.format(EvaluationExceptionKind.FUNCTION_EXECUTION_ERROR.getString()
+                        , methodRef.name()));
+            }
+            if (argValues != null) {
+                return argValues;
+            }
+            List<Value> argValueList = new ArrayList<>();
+            // Evaluates all function argument expressions at first.
+            for (Evaluator argEvaluator : argEvaluators) {
+                argValueList.add(argEvaluator.evaluate().getJdiValue());
+                // Assuming all the arguments are positional args.
+                argValueList.add(EvaluationUtils.make(context, true).getJdiValue());
+            }
+
+            List<Type> types = method.methodRef.argumentTypes();
+            // Removes injected arguments added during the jvm method gen phase.
+            for (int index = types.size() - 1; index >= 0; index -= 2) {
+                types.remove(index);
+            }
+
+            // Todo - IMPORTANT: Add remaining steps to validate and match named, defaultable and rest args
+            // Todo - verify
+            // Here we use the parent strand instance to execute the function invocation expression.
+            Value parentStrand = getParentStrand();
+            argValueList.add(0, parentStrand);
+            return argValueList;
+        } catch (ClassNotLoadedException e) {
+            throw new EvaluationException(String.format(EvaluationExceptionKind.FUNCTION_EXECUTION_ERROR.getString(),
+                    methodRef.name()));
+        }
+    }
```

Comment:
Repeated method mentioned in [here](https://github.com/ballerina-platform/ballerina-lang/pull/26322/files#r505451319)

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/26322#discussion_r505469514